bc_obps/registration/fixtures/operator.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "model": "registration.operator",
+    "pk": 1,
+    "fields": {
+      "id": 1,
+      "legal_name": "ladedah",
+      "trade_name": "ladedah",
+      "cra_business_number": "ladedah",
+      "bc_corporate_registry_number": "ladedah",
+      "business_structure": "ladedah",
+      "mailing_address": "ladedah",
+      "bceid": "ladedah",
+      "parent_operator": 1,
```

Comment:
I know this is dummy data, but it would make more sense not to set this instance as its own parent and compliance obligee.